packages/beacon-node/src/chain/produceBlock/produceBlockBody.ts

```diff
@@ -566,13 +568,19 @@ export async function prepareExecutionPayload(
       feeRecipient: suggestedFeeRecipient,
     });
 
+    const requestStartTime = Date.now() / 1000;
     payloadId = await chain.executionEngine.notifyForkchoiceUpdate(
       fork,
       toRootHex(parentHash),
       safeBlockHash,
       finalizedBlockHash,
       attributes
     );
+    if (ForkSeq[fork] >= ForkSeq.eip7805) {
```

Comment:
this isn't really a focil specific metric anymore and we already track number and duration of fcu calls

<img width="930" height="605" alt="image" src="https://github.com/user-attachments/assets/8a189778-0ce7-4fe8-854c-7624d668df8c" />